src/main/resources/application.properties

```diff
@@ -1,12 +1,11 @@
-# H2 Database Configuration
-spring.datasource.url=jdbc:h2:mem:testdb
-spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=sa
-spring.datasource.password=password
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-spring.h2.console.enabled=true
-spring.h2.console.path=/h2-console
-spring.jpa.defer-datasource-initialization=true
+# PostgreSQL Database Configuration
+spring.datasource.url=jdbc:postgresql://localhost:5432/quickquest
+spring.datasource.username=postgres
+spring.datasource.password=Martiniste#1234
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
‼️ automatically create and drop the database schema on startup and shutdown

'''
Set spring.jpa.hibernate.ddl-auto update
'''